<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetNickName</class>
 <widget class="QDialog" name="SetNickName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nickname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NicknameLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Host ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="HostIpLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hostPort">
       <property name="maximum">
        <number>5000000</number>
       </property>
       <property name="singleStep">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
